array-methods/7-array/index.html

```diff
@@ -9,37 +9,92 @@
   <body>
     <script src="./data.js"></script>
     <script>
-      function countAllPeople() {
-        // your code goes here
-        console.log(got);
+
+      // Go through todo.md before start solving.
+      function countAllPeople(got) {
+        var count=0;
+        for(var i=0;i<got.houses.length;i++) {
+          count+=got.houses[i].people.length;
+        }
+        return count;
       }
 
-      function peopleByHouses() {
+      function peopleByHouses(got) {
         // your code goes here
+        var people1={};
+        got.houses.forEach(house => people1[house.name]= house.people.length );
+        return people1;
       }
```

Comment:
This will not work. `Arrow` function does an implicit return. You can't have an assignment operator there.